modules/core/arrow-test/src/main/kotlin/arrow/test/laws/ReducibleLaws.kt

```diff
@@ -14,53 +14,54 @@ import io.kotlintest.properties.forAll
 
 object ReducibleLaws {
 
-  fun <F> laws(RF: Reducible<F>, cf: (Int) -> Kind<F, Int>, EQ: Eq<Int>, EQOptionInt: Eq<Option<Int>>, EQLong: Eq<Long>): List<Law> =
-    FoldableLaws.laws(RF, cf, EQ) + listOf(
-      Law("Reducible Laws: reduceLeftTo consistent with reduceMap") { RF.reduceLeftToConsistentWithReduceMap(cf, EQ) },
-      Law("Reducible Laws: reduceRightTo consistent with reduceMap") { RF.reduceRightToConsistentWithReduceMap(cf, EQ) },
-      Law("Reducible Laws: reduceRightTo consistent with reduceRightToOption") { RF.reduceRightToConsistentWithReduceRightToOption(cf, EQOptionInt) },
-      Law("Reducible Laws: reduceRight consistent with reduceRightOption") { RF.reduceRightConsistentWithReduceRightOption(cf, EQOptionInt) },
-      Law("Reducible Laws: reduce reduce left consistent") { RF.reduceReduceLeftConsistent(cf, EQ) },
-      Law("Reducible Laws: size consistent") { RF.sizeConsistent(cf, EQLong) }
-    )
+  fun <F> laws(RF: Reducible<F>, G: Gen<Kind<F, Int>>, EQ: Eq<Int>, EQOptionInt: Eq<Option<Int>>, EQLong: Eq<Long>): List<Law> =
```

Comment:
`GenK` + `EqK`. The eq instances for int, option of int and long are also not needed